@noun-auction/components/AuctionUi/NounsBidForm.tsx

```diff
@@ -53,15 +53,15 @@ export function NounsBidForm({
     daoConfig: { auctionContractAddress, abi },
     tokenId,
     contract: { minBidIncrementPercentage, reservePrice },
-    auctionData,
+    activeAuction,
   } = useNounishAuctionProvider()
 
-  if (!abi || !auctionContractAddress) return null
+  if (!abi || !auctionContractAddress || !activeAuction) return null
```

Comment:
This conditional is probably making the rules-of-hooks alarm bells go off.

Reply:
10-4 - removed it - doesn't need to be there.